Patch to add which version of Apache is supported.  This patch has not
yet been submitted upstream.

--- puppetlabs-apache-1.8.1/manifests/version.pp.orig	2016-02-09 08:50:27.000000000 -0800
+++ puppetlabs-apache-1.8.1/manifests/version.pp	2016-04-20 12:56:54.113346691 -0700
@@ -1,3 +1,7 @@
+#######################################################################
+# Oracle has modified the originally distributed contents of this file.
+#######################################################################
+
 # Class: apache::version
 #
 # Try to automatically detect the version by OS
@@ -38,6 +42,9 @@
     'Suse': {
       $default = '2.2'
     }
+    'Solaris': {
+      $default = '2.4'
+    }
     default: {
       fail("Class['apache::version']: Unsupported osfamily: ${::osfamily}")
     }
